src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt

```diff
@@ -77,6 +77,13 @@ val RsFunction.returnType: Ty get() {
 
 val RsFunction.abi: RsExternAbi? get() = externAbi ?: (parent as? RsForeignModItem)?.externAbi
 
+val RsFunction.isActuallyUnsafe: Boolean
```

Comment:
I'm not sure about this name

Reply:
I can't come up with a better name, so let's add javadocs for this property with some details when it's not the same with `RsFunction.isUnsafe`

Reply:
Should we introduce the same property for constants?

Reply:
> Should we introduce the same property for constants?

Constants can't be unsafe

Reply:
Yeah, we can't use const in extent blocks but we can use static variables
```rust
extern {
    static foo: i32;
}
```
And psi element for constant and static variables is the same - `RsConstant`
Is it possible that using of static variable is safe in some cases?

Reply:
Mmm, let's make it in another PR.